packages/mobile/ios/celo/AppDelegate.m

```diff
@@ -152,4 +170,9 @@ - (void)applicationWillEnterForeground:(UIApplication *)application
   self.blurView = nil;
 }
 
+- (void)userNotificationCenter:(UNUserNotificationCenter* )center willPresentNotification:(UNNotification* )notification withCompletionHandler:(void (^)(UNNotificationPresentationOptions options))completionHandler 
+{
+  completionHandler(UNAuthorizationOptionAlert | UNAuthorizationOptionBadge | UNAuthorizationOptionSound);
+}
```

Comment:
If I understand correctly, this will now show iOS notifications while the app is in foreground.
We should then disable our own in-app banner. Right?

Reply:
hm, by in-app banner do you mean the red error and blue informational ones? I don't think they serve the same purpose as the notifications right?